pkgs/development/python-modules/setuptools-gettext/default.nix

```
@@ -12,17 +11,16 @@
 
 buildPythonPackage rec {
   pname = "setuptools-gettext";
-  version = "0.1.11";
+  version = "0.1.14";
   pyproject = true;
 
-  # >=3.12 doesn't work because the package depends on `distutils`
-  disabled = pythonOlder "3.7" || pythonAtLeast "3.12";
+  disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/breezy-team/setuptools-gettext/blob/0e9965aa50eeacd104f08124527b69e827048da6/pyproject.toml#L24

---

Good catch, thanks